src/osx/cocoa/window.mm

```diff
@@ -2153,6 +2185,83 @@ When dragging the bottom part of the DND sample ("Drag text from here!")
     }
 }
 
+void wxWidgetCocoaImpl::TouchesCancel(WX_NSEvent event)
+{
+    if ( wxCocoaGesturesImpl* gestures = wxCocoaGestures::FromObject(this) )
+        gestures->TouchesCancel(event);
+}
+
+void wxCocoaGesturesImpl::TouchesCancel(NSEvent* event)
+{
+    RawTouchEvent(event, wxEVT_TOUCH_CANCEL, NSTouchPhaseCancelled);
+}
+
+void* wxCocoaGesturesImpl::GetTouchID(NSObject* id, wxEventType type)
+{
+    for(size_t i = 0; i < m_touchIDs.size(); i++)
```

Comment:
Here too I'd use range for:
```suggestion
    for ( auto other : m_touchIDs )
```

(the name `other` is confusing to me, so perhaps it would be better to rename it too)

Reply:
`i` is used for return value.